keep/api/routes/workflows.py

```diff
@@ -491,6 +491,50 @@ def get_raw_workflow_by_id(
             )
         },
     )
+    
+@router.get("/{workflow_id}/data", description="Get workflow by ID")
```

Comment:
```suggestion
@router.get("/{workflow_id}", description="Get workflow by ID")
```

Let's use the route "/{workflow_id}" for general info about workflow and change current "/{workflow_id}" to "/{workflow_id}/runs"